drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis
+  class AutoExitJob < BaseJob
+    def perform
+      Hmis::Hud::Project.all.each do |project|
```

Comment:
```suggestion
      Hmis::Hud::Project.hmis.each do |project|
```
This will limit scope to only projects in HMIS data sources